README.md

```diff
@@ -1725,6 +1725,7 @@ _See also [Documentation Generators](#documentation-generators), [Wikimatrix](ht
 - [Awesome Public Datasets](https://github.com/awesomedata/awesome-public-datasets) - List of high quality, topic-centric public data sources.
 - [Awesome Sysadmin](https://github.com/awesome-foss/awesome-sysadmin) - Curated list of amazingly awesome open source sysadmin resources.
 - Lists of software aimed at privacy and decentralization in some form: [PRISM Break](https://prism-break.org/en/), [privacytools.io](https://www.privacytools.io/), [Alternative Internet](https://redecentralize.github.io/alternative-internet/), [Libre Projects](http://libreprojects.net/)
+- [Easy Indie App](https://easyindie.app) - Apps that can be self-hosted in a few clicks.
```

Comment:
Please put this under Self-hosting Solutions

Please include the source code link, license and language.

Reply:
I didn't put it there because I thought it isn't a self-hosting solution (it only points to other ones, like an information portal). Would I need to say `Various` `Various` for license and language?

Reply:
My apologies, I glanced at the GitHub repo earlier and it made me think it was a self hosting solution. This belongs as an external link.